libs/shared/data/src/lib/workspace/target.ts

```diff
@@ -0,0 +1,11 @@
+import { JsonValue } from '@angular-devkit/core';
+
+export interface Target {
```

Comment:
are you sure you can't take the interface from Angular?
https://github.com/angular/devkit/blob/f32607e3cc625e7503147b0334e2a76c43340aca/packages/angular_devkit/architect/src/architect.ts#L95

Reply:
I can, but I don't want to pull it into the FE

Reply:
I'm not sure why not (?)